src/store/types.ts

```diff
@@ -0,0 +1,59 @@
+export const SET_USER = 'SET_USER';
```

Comment:
These could be a `type`, for example: `type UserAction = 'SET_USER' | 'SIGN_OUT'`

Reply:
Something you said that you were going to look into.